grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -132,6 +133,66 @@ class RouteInfo(NamedTuple):
   },
 }
 
+rdf_types_schemas: Dict[str, Schema] = {
+  "RDFDatetime": {
+    "type": "string",
+    "format": "uint64",
+    "description": "RDF type is `RDFDatetime` and it represents "
```

Comment:
I find this _"RDF type is `X` and it represents Y"_ to be quite cumbersome. Why not just _"Y"_ and move the prefix to where you actually generate the message?

Reply:
Thank you! I moved the prefix mentioning RDF type name in `_AddRDFTypesSchemas` so it can be added to the OpenAPI components descriptions as well, not only to the description of fields that have this type.